tools/data/ava/extract_rgb_frames_ffmpeg.sh

```diff
@@ -38,7 +38,7 @@ do
   out_video_dir=${OUT_DATA_DIR}/${video_name}/
   mkdir -p "${out_video_dir}"
 
-  out_name="${out_video_dir}/${video_name}_%06d.jpg"
+  out_name="${out_video_dir}/${video_name}_%05d.jpg"
```

Comment:
Why change this line?

Reply:
Compare with the docs folder strcture, I think the template of file name may be img_%05d.jpg rather than ${video_name}_%05d.jpg

Reply:
Sorry for the mistake, we updated the wrong filename.

Reply:
> Why change this line?

The preprocessing part is not consistent with the implementation (file names of images are different).